crates/cxx-qt-gen/src/parser/parameter.rs

```diff
@@ -14,3 +14,48 @@ pub struct ParsedFunctionParameter {
     /// The name of the C++ type if one has been specified
     pub cxx_type: Option<String>,
 }
+
+impl ParsedFunctionParameter {
```

Comment:
same here, can we split this into another commit then merge separately ?

Reply:
Done